static/src/stylesheets/head.content.scss

```diff
@@ -24,6 +24,7 @@
 @import 'module/_chapters';
 @import 'module/content/_article-immersive';
 @import 'module/content/_article-minute';
+@import 'module/content/_article-labour-liverpool';
```

Comment:
this feels like the CSS should be inlined like the story package css on facia - i.e. not included in general content head css, which is really too big already...